src/fu-engine.c

```diff
@@ -8310,6 +8313,19 @@ fu_engine_load(FuEngine *self, FuEngineLoadFlags flags, FuProgress *progress, GE
 	if (!fu_engine_update_devices_file(self, &error_json_devices))
 		g_info("failed to update list of devices: %s", error_json_devices->message);
 
+	/* add something for coverage */
+	if (g_getenv("CI") != NULL) {
```

Comment:
I'm not really a fan of doing this in core engine code.  Why not use the environment variable to set a directory of attributes?

Reply:
We have `installed-tests/fwupd/tests/bios-attrs/` preinstalled to installed-tests directory...

Reply:
I've moved this bit to https://github.com/fwupd/fwupd/pull/8444